ICAFIX/README.md

```diff
@@ -43,6 +43,15 @@ your compute environment. In particular, the `${FSL_FIXDIR}/settings.sh` file
 likely needs modification. (The settings.sh.WUSTL_CHPC2 file in this directory
 is the settings.sh file that is used on the WUSTL "CHPC2" cluster).
 
+# AutoReclean python dependencies
+
+To create a conda environment for the ApplyAutoReclean pipeline, use this command:
+
+```bash
+conda env create -f "$HCPPIPEDIR"/environment.yml [-n AutoRecleanEnv or -p /somewhere/you/want/the/conda/environment/installed ]
+```
+If using the native python version, please specify `--python-interpreter` with its absolute path, such as `--python-interpreter=/usr/bin/python3`.
+
```

Comment:
I think something went wrong with the rebase, there shouldn't be changes to these folders in the PR.